Library/DasharoSystemFeaturesUiLib/DasharoSystemFeatures.c

```diff
@@ -743,6 +748,68 @@ DasharoSystemFeaturesUiLibConstructor (
     ASSERT_EFI_ERROR (Status);
   }
 
+  BufferSize = sizeof(mDasharoSystemFeaturesPrivate.DasharoFeaturesData.CpuThrottlingThreshold);
+  Status = gRT->GetVariable (
+      mCpuThrottlingThresholdEfiVar,
+      &gDasharoSystemFeaturesGuid,
+      NULL,
+      &BufferSize,
+      &mDasharoSystemFeaturesPrivate.DasharoFeaturesData.CpuThrottlingThreshold
+  );
+
+  if (Status == EFI_NOT_FOUND) {
+    mDasharoSystemFeaturesPrivate.DasharoFeaturesData.CpuThrottlingThreshold = mCpuThrottlingThresholdDefault;

```

Comment:
These are static values, they don;t need to be saved to / read from EFI variables

Reply:
Removed unneeded efivars in 6e30d147a2d5d42f557519cd91dea5059fcdf7ff